numpy/core/src/common/simd/vec/operators.h

```diff
@@ -144,8 +144,13 @@ NPY_FINLINE npyv_f64 npyv_not_f64(npyv_f64 a)
 // ANDC, ORC and XNOR
 #define npyv_andc_u8 vec_andc
 #define npyv_andc_b8 vec_andc
-#define npyv_orc_b8 vec_orc
-#define npyv_xnor_b8 vec_eqv
+#ifdef NPY_HAVE_VX
+    #define npyv_orc_b8(A, B) npyv_or_b8(npyv_not_b8(B), A)
+    #define npyv_xnor_b8(A, B) npyv_not_b8(npyv_xor_b8(B, A))
+#else
+    #define npyv_orc_b8 vec_orc
+    #define npyv_xnor_b8 vec_eqv
+#endif
```

Comment:
I would think this works like (best guess, I don't have anything other than wikipedia to go by)
arch | defines
---|---
v13 | `NPY_HAVE_VX`
v14 | `NPY_HAVE_VX`, `NPY_HAVE_VXE`
v15 | `NPY_HAVE_VX`, `NPY_HAVE_VXE`, `NPY_HAVE_VXE2`

If that is correct, than this will always use the first clause

Reply:
@mattip : Thanks for pointing out. I have updated the changes. Please let me know if you have any comments.